.travis.yml

```diff
@@ -4,7 +4,7 @@ cache:
     - $HOME/.cache/pip
 language: python
 python:
-  - "2.6"
+  - "2.7"
```

Comment:
@rhelmer I just noticed that 
```
docs/development/build-pkg-from-source.rst
docs/development/codeanddatabaseupdate.rst
docs/development/dev-services.rst
```
Also speak of Python 2.6. 
Can I just blanket update them too?